src/TaskManager/TaskManager/appsettings.Local.json

```diff
@@ -1,10 +1,11 @@
 {
   "WorkloadManagerDatabase": {
-    "ConnectionString": "mongodb://root:rootpassword@localhost:30017",
```

Comment:
is it possible to leave this file as is ? or go through and update the developer-setup accordingly :)